Create page: update intro section (#2803)
home-assistant/developers.home-assistant: docs/documenting/create-page.md

title: "Create a new page"
---

To create a new integration page, follow these steps:

> 1. The fastest way is to make a copy of the [integration documentation template](https://github.com/home-assistant/home-assistant.io/tree/current/source/_integrations/_integration_docs_template.markdown) and edit it.

Reviewer: ```suggestion
1. Make a copy of the [integration documentation template](https://github.com/home-assistant/home-assistant.io/tree/current/source/_integrations/_integration_docs_template.markdown) and edit it.
```

2. Make sure the filename of the integration page matches the domain name of the integration.

> 3. The [Integration overview](https://www.home-assistant.io/integrations/) and the [Examples section](https://www.home-assistant.io/cookbook/) are generated automatically, so there is no need to add a link to those pages.

Reviewer: this one is a bit confusing as a "step". maybe make it a sub bullet point on step 1, or as a note outside the steps?

Additional keys for the file header:

- `title`: This title should match with the name of the integration as written in the integration manifest file.

> - `ha_release`: The release when the integration was included. For example, "0.2025.9". If the current release is 2025.8, make `ha_release` 2025.9. If it end in a `0`, as in 2025.10, quote it with `' '`, otherwise the zero won't be displayed.

Reviewer: ```suggestion
- `ha_release`: The release when the integration was included. If the current release is 2025.8, make `ha_release` 2025.9. If it ends in a `0`, as in 2025.10, quote it with `' '`, otherwise the zero won't be displayed.
```
the example is already in the following sentence. If you prefer to keep it, remove the prefix `0.`

Additional keys for the file header:

- `title`: This title should match with the name of the integration as written in the integration manifest file.
- `ha_release`: The Home Assistant release when the integration was included.
  - If the current release is 2025.8, make `ha_release` 2025.9.

>   - If it ends in a `0`, as in '2025.10', quote it with `' '`, otherwise the zero won't be displayed.

Author: ```suggestion
  - For the October release, as in '2025.10', quote it with `' '`, otherwise the zero won't be displayed.
```
